packages/admin/src/pages/technologies/Form/AttachmentsForm.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import { Card, CardContent } from '@material-ui/core';
+import {
+	useQuery,
+	Loading,
+	Error,
+	SimpleShowLayout,
+	TextField,
+	ImageField,
+	ArrayField,
+	Datagrid,
+	UrlField,
+	DateField,
+} from 'react-admin';
+
+const useStyles = makeStyles({
+	img: {
+		width: 200,
+		height: 200,
+		'& img': {
+			margin: 0,
+			width: '100%',
+			height: '100%',
+		},
+	},
+	cardStyle: {
+		maxWidth: 250,
+		margin: '0.2em',
+		display: 'inline-block',
+		verticalAlign: 'top',
+	},
+});
+
+const AttachmentsForm = ({ record, resource, basePath }) => {
+	const classes = useStyles();
+	const { loading, error, data } = useQuery({
+		type: 'getList',
+		resource: `uploads`,
+		payload: {
+			filter: { object_id: record.id },
+			pagination: {
+				page: 1,
+				perPage: 100,
+			},
+			sort: {
+				field: 'id',
+				order: 'asc',
+			},
+		},
+	});
+	if (error) return <Error />;
+	if (loading) return <Loading />;
+
+	// eslint-disable-next-line no-shadow
+	const Cards = ({ record, source }) => {
+		return record[source].map(({ id, url }) => {
+			return (
+				<Card key={id} className={classes.cardStyle}>
+					<CardContent>
+						<ImageField
+							record={{ url }}
+							source="url"
+							title="url"
+							className={classes.img}
+						/>
+					</CardContent>
+				</Card>
+			);
+		});
+	};
+
+	const newRecord = {
+		imagens: data.filter(({ filename }) => !filename.match(/.pdf/g)),
```

Comment:
```suggestion
		images: data.filter(({ filename }) => !filename.match(/.pdf/g)),
```

What about fixing this? 😏

Reply:
@luizeboli I didn't even see that typo! Nice eyesight :laughing:

Reply:
It's the habit of writing in portuguese. Sometimes I do that too 😅